kernel/arch/dreamcast/hardware/maple/keyboard.c

```diff
@@ -4,7 +4,11 @@
    Copyright (C) 2002 Megan Potter
    Copyright (C) 2012 Lawrence Sebald
    Copyright (C) 2018 Donald Haase
+<<<<<<< HEAD
```

Comment:
tsk tsk

Reply:
OH NOEZ YOU GOT ME!!! I'm literally working on it right now and was like "WTF is this!?!"